include/controller/cpp/webots/Node.hpp

```diff
@@ -130,6 +130,8 @@ namespace webots {
     Field *getProtoField(const std::string &fieldName) const;
     const double *getPosition() const;
     const double *getOrientation() const;
+    const double *getPose() const;
+    const double *getPose(const Node *fromNode) const;
```

Comment:
Isn't that simpler?
```suggestion
    const double *getPose(const Node *fromNode = NULL) const;
```

Reply:
Unfortunately no because SWIG gets confused and doesn't know how to make Python and JAVA interface with default values. Alternatively, I write a custom code in `controller.i` but then it is not consistent with `worldSave`